abstract.tex

```diff
@@ -0,0 +1,79 @@
+\documentclass{anstrans}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\title{Updating PyRe for the Current Cyclus Framework}
+\author{Rhys R. MacMillan,$^{*}$ Nathan S. Ryan,$^{*}$ and Kathryn D. Huff$^{*}$}
+
+\institute{
+$^{*}$Advanced Reactors and Fuel Cycles Group, University of Illinois, Urbana IL, rhysrm2@illinois.edu
+}
+
+%%%% packages and definitions (optional)
+\usepackage{graphicx} % allows inclusion of graphics
+\usepackage{booktabs} % nice rules (thick lines) for tables
+\usepackage{microtype} % improves typography for PDF
+\usepackage{xspace}
+\usepackage{float}
+
+\newcommand{\SN}{S$_N$}
+\renewcommand{\vec}[1]{\bm{#1}} %vector is bold italic
+\newcommand{\vd}{\bm{\cdot}} % slightly bold vector dot
+\newcommand{\grad}{\vec{\nabla}} % gradient
+\newcommand{\ud}{\mathop{}\!\mathrm{d}} % upright derivative symbol
+\newcommand{\cycamore}{\textsc{Cycamore}\xspace}
+\newcommand{\cyclus}{\textsc{Cyclus}\xspace}
+
+\begin{document}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Introduction}
+The United States has exclusively operated a once-through civilian fuel cycle. 
+Over several decades, the country’s many reactors have generated a stockpile of used fuel. Interest in closing the fuel cycle by reprocessing is increasing so the volume of high level waste that requires storage is reduced, and the valuable fissionable isotopes mixed into that waste won’t be discarded. Reprocessing nuclear fuel does come with some challenges, with proliferation concerns being high on the list for the U.S. government. To deal with these concerns, researchers at Argonne National Laboratory developed pyroprocessing, which is a reprocessing method that uses electrochemical separation and doesn’t produce a plutonium material stream. 
```

Comment:
```suggestion
Over several decades, the country’s many reactors have generated a stockpile of used fuel. Interest in closing the fuel cycle by reprocessing is increasing so the volume of high-level waste that requires storage is reduced, and the valuable fissionable isotopes mixed into that waste won’t be discarded. To deal with proliferation concerns from reprocessing, researchers at Argonne National Laboratory developed pyroprocessing, which is a reprocessing method that uses electrochemical separation and doesn’t produce a plutonium material stream. 
```
Your second sentence is passive.
This should also set up Greg's incentive for creating PyRE. Talk a little more about diversion detection and identifying diversion pathways using detector efficiencies.